src/components/PopupRevisionOverview/PopupRevisionOverview.js

```diff
@@ -570,10 +583,12 @@ const ChangeContainer = ({
 }
 
 /**
- * @param {Object} props
- * @param {Object} props.originalObject - Contains the object in its original form
- * @param {Object} props.changesObject - Contains the object with changes on the properties (e.g. Propertie: {"new": [...], "removed": [...], "same": [...]})
- * @returns Component that displays the changes of the GEO property on a Leaflet map
+ * Component that renders the RevisionWerkingsgebied component, which displays the revisie werkingsgebied on a map.
```

Comment:
> Component that renders the RevisionWerkingsgebied component

This is unnecessary, because we assume that a React Component will always return itself to be rendered.

Reply:
Good point, I've removed the component description "renders the RevisionWerkingsgebied component".